codegen/smithy-dafny-codegen/build.gradle.kts

```diff
@@ -1,6 +1,8 @@
 // Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import java.util.regex.Pattern.compile
+
```

Comment:
Suggestion: Remove unused import. 
Presumably this was for a since abandoned idea. 
```suggestion
```